frontend/containers/AuctionsContainer.vue

```diff
@@ -80,6 +81,9 @@ export default Vue.extend({
         hasAcceptedTerms(): boolean {
             return this.$store.getters['preferences/getAcceptedTerms'];
         },
+        isStagingEnvironment() {
+            return process.env.IS_STAGING_ENVIRONMENT;
+        },
```

Comment:
You don't have to pass it all the way from the container, especially if don't add a story to display behaviour of this prop

Reply:
So I can just add this computed property into the MainFlow component directly?

Reply:
Yes, I think it's ok